Dockerfile

```diff
@@ -6,13 +6,25 @@ VOLUME /output
 RUN mkdir -p /jenkins_node_scanner
 WORKDIR /jenkins_node_scanner
 
+RUN apt-get update \
+    && apt-get install -y supervisor=3.3.5-1 wget=1.20.1-1.1 --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN wget -q -O - https://github.com/papertrail/remote_syslog2/releases/download/v0.17/remote_syslog_linux_amd64.tar.gz \
+    | tar -zxf -
+
+
 COPY Pipfile /jenkins_node_scanner
 COPY Pipfile.lock /jenkins_node_scanner
+COPY jenkins_node_scanner.py /jenkins_node_scanner
```

Comment:
Why did this line move up here? Let's move it back to be next to the other `COPY` statements.

Reply:
Ah my bad! I thought it would make more sense for that line to go there since I didn't think it was Papertrail specific. Thanks!